README.md

```diff
@@ -150,4 +151,4 @@ Totally worth to check:
 - https://github.com/sovereign/sovereign
 - https://roll.urown.net/about.html
 
-Cover art pirated from [KADA★BURA](https://www.kadaburadraws.com/pixel-art#/text-rpg/)
\ No newline at end of file
+Cover art pirated from [KADA★BURA](https://www.kadaburadraws.com/pixel-art#/text-rpg/)
```

Comment:
I don't really know what this change is about, it seems to be introduced by the GitHub editor for reasons I don't fully understand.